Sources/Commands/SwiftBuildTool.swift

```diff
@@ -64,24 +64,24 @@ private enum BuildToolFlag: Argument {
     case xcc(String)
     case xld(String)
     case xswiftc(String)
-    case buildPath(String)
+    case buildPath(AbsolutePath)
     case buildTests
-    case chdir(String)
+    case chdir(AbsolutePath)
     case colorMode(ColorWrap.Mode)
     case ignoreDependencies
     case verbose(Int)
-    case xcconfigOverrides(String)
+    case xcconfigOverrides(AbsolutePath)
 
     init?(argument: String, pop: () -> String?) throws {
 
         func forcePop() throws -> String {
             guard let value = pop() else { throw OptionParserError.expectedAssociatedValue(argument) }
             return value
         }
-
+        
         switch argument {
         case Flag.chdir, Flag.C:
-            self = try .chdir(forcePop())
+            self = try .chdir(AbsolutePath(forcePop()))
```

Comment:
Ah, yes indeed.  I had tested with an absolute path `-C` argument (for example) but not relative.